syntax = "proto3";

package kvstore;

service KVStore {
  rpc MyMethod (MyRequest) returns (MyResponse) {}
}

message MyRequest {
  string name = 1;
}

message MyResponse {
  string message = 1;
}